test/jdk/java/time/tck/java/time/TCKOffsetDateTime.java

```diff
@@ -1381,6 +1381,7 @@ public void test_compareTo_hourDifference() {
         assertEquals(a.compareTo(a) == 0, true);
         assertEquals(b.compareTo(b) == 0, true);
         assertEquals(a.toInstant().compareTo(b.toInstant()) == 0, true);
+        assertEquals(OffsetDateTime.timeLineOrder().compare(a,b) == 0, true);
```

Comment:
Could be in a separate test case, as this method tests `compareTo` method?

Reply:
This is one of a sequence of tests of compareTo.  The other (previous) tests include verifying the assertion about the result of the `timeLineOrder` comparator with the same data.